src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -422,9 +441,47 @@ def build_purpose_or_feature_section_and_update_vendor_map(
                 privacy_declaration_row=privacy_declaration_row,
             )
 
+        # Separately, add AC Vendor to Vendor Consent Map if applicable
+        add_ac_vendor_to_vendor_consent_map(
+            vendor_map=vendor_map,
+            tcf_vendor_component_type=tcf_vendor_component_type,
+            privacy_declaration_row=privacy_declaration_row,
+        )
+
     return non_vendor_record_map, vendor_map
 
 
+def add_ac_vendor_to_vendor_consent_map(
+    vendor_map: Dict[str, VendorRecord],
+    tcf_vendor_component_type: VendorSectionType,
+    privacy_declaration_row: Row,
+) -> None:
+    """
+    Add systems with ac.*-prefixed vendor records to the Vendor Consent section if applicable.
+
+    FE shows Consent toggle only for AC vendors, and they are not required to have Privacy Declarations
+    """
+    vendor_id, system_identifier = get_system_identifiers(privacy_declaration_row)
```

Comment:
nit: do we even want to support `system_id` specifying an AC vendor? not that it's causing any real harm, and it could potentially leave things more flexible. but i do feel like the flexibility and ambiguity here could potentially cause confusion going forward, and i'm inclined to just be a little bit more restrictive and say `vendor_id` is what we care about, and that's that -- at least within the scope of AC support.

more broadly, i know we'd kept system_id/vendor_id open ended in the past for good reason - maybe there are some other points in the overlay generation where we could start to hone in on the vendor_Id and remove use of the system_id. that's not within scope of this PR, and if you think that's not a good idea then no need to follow up on it. 

the point of this comment was regarding the AC use case specifically and i just thought it was worth mentioning that broader thought, without doing much to check whether it makes sense at this point! 😄

Reply:
that's a good callout.  I'll restrict so we're ignoring if there's no vendor id.